setup.py

```diff
@@ -81,6 +81,8 @@ def run(self):
             "ipyparallel",
             "pre-commit",
             "pytest-timeout",
+            "gobject",
```

Comment:
i think `pygobject` is the right pip name:
- https://pypi.org/project/PyGObject/
- https://pypi.org/project/gobject/